packages/app/package.json

```diff
@@ -99,7 +103,7 @@
     "@types/jest": "^28.1.6",
     "@types/lodash": "^4.14.186",
     "@types/pluralize": "^0.0.29",
-    "@types/react": "18.2.23",
+    "@types/react": "^18.2.0",
```

Comment:
this seems to cause issue on unit test. any reason for this change ?

Reply:
interesting, its a pretty common TS/React error when the type doesn't match the react package, but let me see if I can handle in a different way, give me a moment